src/components/ui/Tags/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { View } from "react-native";
+import { Chip, ChipProps } from "react-native-elements";
+import { RFValue } from "react-native-responsive-fontsize";
+import { colors } from "@app/utils";
+import styles from "./styles";
+
+interface Tag extends ChipProps {
+  id?: string;
+  title: string;
+}
+
+interface TagsProps {
+  tags: Tag[];
+  id?: string;
+  variant?: "round" | "square";
+  color?: "teal" | "lightbrown" | "orange";
+}
+
+const Tags: React.FC<TagsProps> = ({ tags, id, variant = "round", color = "orange" }) => {
```

Comment:
Can we use defaultProps for prop types with default values?

https://blog.bitsrc.io/understanding-react-default-props-5c50401ed37d

Reply:
For functional components, I'd say defaultProps is outdated and may potentially? get deprecated in the future
https://stackoverflow.com/questions/41488537/react-defaultprops-vs-es6-default-params-when-destructuring-performances-issu